SpatialGDK/Source/SpatialGDKFunctionalTests/Public/GeneratableTestMap.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "GeneratableTestMap.generated.h"
+
+UINTERFACE(MinimalAPI)
+class UGeneratableTestMap : public UInterface
+{
+	GENERATED_BODY()
+};
+
+class IGeneratableTestMap
+{
+	GENERATED_BODY()
+
+public:
+	virtual void GenerateMap() { check(false); }
+
+	virtual bool SaveMap()
+	{
+		check(false);
+		return false;
+	}
```

Comment:
```suggestion
	virtual bool SaveMap() = 0;
```
Is the usual way of implementing abstract / interface classes.

Reply:
I assume this applies to all the other functions in this class?

Reply:
Yeah sorry my suggestion was for all functions, should have made that clear.